AMW_angular/io/src/app/resources/resource-type-delete/resource-type-delete.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="container">
+  <app-modal-header [title]="'Delete resource type'" (cancel)="cancel()"></app-modal-header>
+  <div class="modal-body">
+    <div class="form-horizontal mx-2">
+      <div class="mb-3">
+        <div class="text-body">Do you really want to delete the resource type {{ resourceType.name }}?</div>
```

Comment:
Add additional info for user similar to old ui:
`(This will also remove all instances as well as sub resource types)`